rpc/grpc/client/block_results_service.go

```diff
@@ -0,0 +1,64 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	abci "github.com/cometbft/cometbft/abci/types"
+	cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
+
+	"github.com/cosmos/gogoproto/grpc"
+
+	brs "github.com/cometbft/cometbft/proto/tendermint/services/block_results/v1"
+)
+
+type ResultBlockResults struct {
```

Comment:
Does it not make sense to call this something like `BlockResults`? Seems simpler.